src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -254,7 +251,7 @@ public class LDAPSecurityRealm extends AbstractPasswordBasedSecurityRealm {
 
     /**
      * The root DN to connect to. Normally something like "dc=sun,dc=com"
-     *
+     * <p>
```

Comment:
tbh honest, even if non failing. I don't like those non closed elements :) 
maybe we should have the checsktyle rule "Tight-HTML" to enforce that.

Reply:
I don't have a preference on this, we can use `<p>…</p>` too.